pkg/machine/qemu/machine_unix.go

```diff
@@ -33,6 +35,18 @@ func checkProcessStatus(processHint string, pid int, stderrBuf *bytes.Buffer) er
 	return nil
 }
 
+func forwardPipeArgs(cmd []string, name string, destPath string, identityPath string, user string) ([]string, error) {
+	return cmd, nil
+}
+
+func podmanPipe(name string) *machine.VMFile {
+	return nil
+}
+
+func useFdVLan() bool {
+	return strings.ToUpper(os.Getenv("CONTAINERS_USE_SOCKET_VLAN")) != "TRUE"
```

Comment:
No objection to having an env flag for experimentation/debugging for now. However, it probably makes sense to drop the fd approach in the long term. Perhaps an intermediate step in a future PR would be to detect qemu < 7.2.0 and leave fd as a fallback. Either way, the more conservative approach this PR takes is good for now.

Reply:
I created a feature request for this functionality https://github.com/containers/podman/issues/18314